lib/system/deploy.rb

```diff
@@ -15,10 +17,32 @@ class Info
 
       def initialize(info)
         @revision = info.fetch('revision') { `git rev-parse HEAD 2> /dev/null`.strip }
-        @release = ENV['AMP_RELEASE'].presence
+        @release = info.fetch('release') { '2.0.0' }
         @deployed_at = info.fetch('deployed_at') { Time.now }
         @error = info.fetch(:error) if info.key?(:error)
       end
+
+      def minor_version
+        version.segments[1]
+      end
+
+      def major_version
+        version.segments[0]
+      end
+
+      private
+
+      class UnknownVersion
+        def segments
+          %w[unknown unknown]
```

Comment:
This means whenever we pass anything that `Gem::Version.new(release)` cannot parse (e.g. "upstream"), we will show "unknown" instead. Maybe better better showing whatever is in "release" properly escaped in this case.